### PR TITLE
Disable upload page buttons after initiated upload

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: npm/src/upload/upload-form.ts [hunk 1]
```diff
@@ -175,6 +175,8 @@ export class UploadForm {
   ) {
     this.formElement.addEventListener("submit", (ev) => {
       ev.preventDefault()
+
+      this.disableButtonsAndDropzone()
```
Comment (suzannehamilton) [inline npm/src/upload/upload-form.ts:179]: Let's have a chat about this in our 1-1 today. I think this change will help in most circumstances, but there are some subtle edge-cases that this might not catch.

Comment (suzannehamilton) [reply npm/src/upload/upload-form.ts:179]: It would also be good to have a test to make sure the form only gets submitted once, but let's chat about the implementation first.

**Comment by suzannehamilton** [comment]
Also temporarily closing this one until Nigel can get back to it to reduce the list of open PRs that need to be checked.

**Review by suzannehamilton — DISMISSED** [review]
🚀

**Review by suzannehamilton — APPROVED** [review]
